src/ocm/components/clusterWizard/ClusterWizardFooter.tsx

```diff
@@ -33,6 +33,7 @@ const ValidationSection = ({
   const { currentStepId } = useClusterWizardContext();
   const validationsInfo = cluster && selectClusterValidationsInfo(cluster);
   const { t } = useTranslation();
+  const allHostsAreVsphere = cluster && cluster.platform?.type === 'vsphere';
```

Comment:
So, in reality it's the cluster that is determined to be "vsphere", not the hosts?
If one host was not in vsphere, would the platform type still be `vsphere`?

Reply:
Is confusing because I only find "vsphere" platform type in clusters. In hosts I can't find this information

Reply:
I've sended the code to a guy that have one vsphere local environment to test it